Updated German localization for 0.10

meandmykeks opened the pull request.

meandmykeks committed:
Fix/Workaround for lua error by Korean localization

Added conditional for unused locale identifier

Added 
	- "Crittershape" (but without Korean string)
	- "Normal" (but without Korean string)
	- "Mythic" (but without Korean string)

Removed
	- "Normal or Heroic"

meandmykeks committed:
Revert "Fix/Workaround for lua error by Korean localization"

This reverts commit 51cf924534feea8a84cb61fb024612fe345b1ef7.

meandmykeks committed:
Merge branch 'christopheml:master' into master

meandmykeks committed:
updated keys for 0.10

Modified:
 - "WARNING_NOT_NIGHT_FAE": fixed line break

Added：
 - "Available since"
 - "Soulshape Journal"
 - "Squirrel Soul"
 - "Squirrel Soul Guide"

christopheml commented:
Hello, there a few new keys for a potential 0.12, you might want to include them in this PR.

meandmykeks committed:
Merge branch 'christopheml:master' into master

meandmykeks committed:
updated keys for 0.12

Added：
 - "BLIZZARD_MAP_PIN_TOOLTIP"
 - "TOMTOM_WAYPOINT_TOOLTIP"
 - "Spectral Feed"
 - "Spectral Feed Tooltip"

meandmykeks commented:
Oh. Didn't see the 0.12 commit was added to this PR automatically... I had already closed it and created a new PR.
